src/components/Layout/Layout.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { NavLink } from 'react-router-dom';
+import links, { logos } from './navData';
+import logo from '../../images/booking.png';
+
+const Layout = ({ children }) => (
+  <div className="row">
+    <nav className="col-md-3 col-lg-3 shadow vh-100">
+      <div className="logo-image">
+        <img src={logo} alt="" />
+      </div>
+      <ul>
+        {links.map((link) => (
+          <li key={link.id} className="link">
+            <NavLink to={link.path}>
+              <span>{link.title}</span>
+            </NavLink>
+          </li>
+        ))}
+      </ul>
+
+      <div className="logos">
+        {logos.map((logo) => (
+          <>{logo}</>
+        ))}
+      </div>
```

Comment:
You made a great effort adding the icons here. However, your map method throws an error in the console since it does not have a unique key.  Please consider adding the icons directly to the file.  For example:
```
<div className="logos">
  <FaIcons.FaTwitter  />
  <FaIcons.FaFacebook  />
  <FaIcons.FaGooglePlus  />
  <FaIcons.FaVimeo  />
  <FaIcons.FaPinterest  />
</div>
```